hack/helm/release/internal/helm/helm.go

```diff
@@ -50,7 +52,12 @@ func Release(conf ReleaseConfig) error {
 	if err != nil {
 		return fmt.Errorf("while creating temp dir: %w", err)
 	}
-	defer os.RemoveAll(tempDir)
+	// keep and print the temp dir in dry run mode
+	if conf.DryRun {
+		fmt.Println("Helm release tempdir:", tempDir)
+	} else {
+		defer os.RemoveAll(tempDir)
+	}
```

Comment:
When would you need that? Isn't that the point of dry-run to begin with?

Reply:
@pebrc dry-run doesn't actually do the release, which is where this would come into play. If you were debugging a failure during the actual release of the charts (which we've absolutely done in the past) and needed the data in the temp directory this would be useful.

Reply:
Yes, I can see the use case even though it should be super rare. I'll add it.